### ISSUE TITLE
Got ProcessFailed on CoreWebView2 when fixed runtime is installed in a folder with Chinese characters 

### ISSUE THREAD

**Issue by xlyang2000** [MicrosoftEdge/WebView2Feedback#2423]
I believe I found a bug with webview2 loaded from a fixed version which is installed in a folder with Chinese characters. In my case, I have the fixed runtime folder installed at  C:\WebView\Microsoft.WebView2.FixedVersionRuntime測試員. 

The calls for CoreWebView2Environment.CreateAsync and EnsureCoreWebView2Async both seem to be successful, but my page can't be loaded with the error of process failed (reason: crash) as shown in the screenshot below. 

Any ideas if this is a known issue?

Thanks!
```
     string path = @"C:\WebView\Microsoft.WebView2.FixedVersionRuntime測試員";
     
      var env2 = await CoreWebView2Environment.CreateAsync(path, "c:\\test2");

      Task task = webView.EnsureCoreWebView2Async(env2);
      await task;

      if (webView.CoreWebView2 != null)
        webView.CoreWebView2.ProcessFailed += CoreWebView2_ProcessFailed;
```

![image](https://user-images.githubusercontent.com/11947286/166978110-184467e3-7bf0-40f6-bd91-93125f577f9e.png)

**Comment by LiangTheDev** [comment]
I could not repro the issue. Could you please share the dump files under c:\test2\EBWebView\Crashpad\reports? That will tell us why it crashed?

**Comment by xlyang2000** [comment]
There is no crash. If you have my code above and put a breakpoint on CoreWebView2_ProcessFailed, you will see it in the debugger. Let me know if you need my VS project.

**Comment by LiangTheDev** [comment]
Is there any files under c:\test2\EBWebView\Crashpad\reports? The screenshot above indicates that GPU process crashed and we normally expect some dump file generated. Also, how many times has the ProcessFailed event fired and with what arguments?

**Comment by xlyang2000** [comment]
There is no application crash, so no crash reports under c:\test2\EBWebView\Crashpad\reports.
But if you put a breakpoint on ProcessFailed, it is being hit 6 times. The argument is in the screenshot 
![image](https://user-images.githubusercontent.com/11947286/167044004-5fe6a855-a3f2-441f-95ef-3e838cca77b7.png)

**Comment by LiangTheDev** [comment]
Just to confirm that all 6 times, the arguments are the same GPU process crashed?
When child webview2 processes like GPU process crashes, WebView2 runtime normally will generate dump files in EBWebView\Crashpad\reports sub folder of the user data folder. That's why I were asking. Are you running Windows 10 and turned off Diagnostic data collection in Windows settings app? (In Settings app, search for diagnostic and ensure that we allow Optional diagnostic data).
It is also possible that the crash happened early and the child process crash are not captured by WebView2 runtime. In this case, could you also check whether there is any Windows Error Report events in the event viewer app, and share the bucket id from the event, or simply share the file %SystemRoot%\System32\Winevt\Logs\Application.evtx?

**Comment by xlyang2000** [comment]
Yes, the arguments are all the same as the screenshot above shows.

**Comment by xlyang2000** [comment]
Here is the setting for diagnostic:
![image](https://user-images.githubusercontent.com/11947286/167052111-1547693e-4ffc-4863-aae2-0cce33fad984.png)

**Comment by xlyang2000** [comment]
Here is the zipped file from %SystemRoot%\System32\Winevt\Logs\Application.evtx 
[log.zip](https://github.com/MicrosoftEdge/WebView2Feedback/files/8637037/log.zip)

**Comment by LiangTheDev** [comment]
There is no event logged in the Application.evtx.
We'll have to ask more questions,
- If you remove the Chinese characters from the fixed runtime folder, does it work?
- Are you running the app elevated? If yes, does it work if you run the app un-elevated?
- Does it work if you run the app with `--edge-webview-switches=--disable-gpu` command line switch, like `WebView2.exe --edge-webview-switches=--disable-gpu`? If it still not working, does it work with `--edge-webview-switches=--in-process-gpu`? If still doesn't work, what is the argument of ProcessFailed event?

**Comment by LiangTheDev** [comment]
The reason that I asked a lot of questions is that I could not repro the issue. And it is strange that we still got GPU process exited failure with --in-process-gpu as there should be no GPU process anymore.

**Comment by xlyang2000** [comment]
do you want me to send you the project? Or the complete code so that you can build on your end?

**Comment by LiangTheDev** [comment]
I don't think that other part of the project matters, if the only difference between working and not working is whether the browser executable folder has Chinese characters in it. It should be some other environment differences that makes it repro for you but not for me. We have to think more and figure out what that is.

**Comment by xlyang2000** [comment]
Ok. Thanks.

**Comment by xlyang2000** [comment]
@LiangTheDev Hi, some updates here. I found that both options edge-webview-switches=--in-process-gpu and --edge-webview-switches=--disable-gpu work for this case. Do you have any ideas which option we should use? Are they not turned on be default? Thanks.
